doc/source/algorithms.rst

```diff
@@ -163,6 +163,7 @@ The following is an example that shows how to use this function:
 The result has several fields, among which the following are most useful:
 
 * ``parents[i]``:  the parent vertex of the i-th vertex. The parent of each source vertex is itself.
+* ``hasparent[i]``:  ``true`` if the i-th vertex has a parent, and ``false`` otherwise. When ``hasparent[i] == false``, it means that the vertex at index ``i`` isn't reachable from any source. Note that ``hasparent[i] == true`` for all source vertices.
```

Comment:
let's continue the discussion over at https://github.com/JuliaLang/Graphs.jl/issues/150, just to make sure that there aren't other users whose code depends on this behavior